contracts/upgradeability/EternalStorage.sol

```diff
@@ -14,4 +14,12 @@ contract EternalStorage {
     mapping(bytes32 => bool) internal boolStorage;
     mapping(bytes32 => int256) internal intStorage;
 
+
+    mapping(bytes32 => uint256[]) internal uintArrayStorage;
```

Comment:
Is there any use case for these new Array mappings? If not, I think we should remove them

Reply:
I use the mapping `addressArrayStorage` for the `initialValidators` in the factories.